Main_project_Tests_TestUnit.rb

```diff
@@ -15,134 +15,187 @@ def setup
  		@driver.navigate.to 'http://demo.redmine.org/'
  	end
 
-	# def test_register_user
-	# 	user = register_user({})
-	# 	expected_text = 'Your account has been activated. You can now log in.'
-	# 	actual_text = @driver.find_element(:css, '#flash_notice').text
-	# 	assert_equal(expected_text, actual_text)
-	# end
-
-	# def test_log_out
-	#   	register_user({})
-	#    	log_out
-	#  	assert(@driver.find_element(:css, '.login').displayed?)
-	# end
-
-	# def test_log_in
-	#    	user = register_user({})
-	#    	log_out
-	#    	log_in(user)
-
-	#   	expected_login = user.login
-	#   	actual_login = @driver.find_element(:css, '[class="user active"]').text
-	#    	assert_equal(expected_login, actual_login)
-	# end
-
- # 	def test_change_pass
- # 		user = register_user({})
- # 		change_pass(user)
- # 		expected_notice = 'Password was successfully updated.'
- # 		actual_notice = @driver.find_element(:css, '#flash_notice').text
- # 		assert_equal(expected_notice, actual_notice)
- # 		log_out
- # 		log_in(user)
- # 		expected_login = user.login
-	# 	actual_login = @driver.find_element(:css, '[class="user active"]').text
-	#  	assert_equal(expected_login, actual_login)
- # 	end
-
-
- # 	def test_create_new_project
- # 		user = register_user({})
- # 	 	project = create_new_project({})
-
- # 	 	expected_notice = 'Successful creation.'
- # 	 	actual_notice = @driver.find_element(:css, '#flash_notice').text
- # 	 	assert_equal(expected_notice, actual_notice)
- # 	end
-
- #  	def test_create_new_version
- #   	 	user = register_user({})
- #   	 	project = create_new_project({})
- #   		create_new_version(project)
-
- #   		expected_notice = 'Successful creation.'
- #   	    actual_notice = @driver.find_element(:css, '#flash_notice').text
- #  	 	assert_equal(expected_notice, actual_notice)
- #   	end
-
-	# def test_create_bug
-	#     user = register_user({})
-	#     project = create_new_project({})
-	#     issue = create_new_issue({type: '1'})
+
+	def test_register_user
+		user = register_user({})
+		
+		expected_text = 'Your account has been activated. You can now log in.'
+		actual_text = @driver.find_element(:css, '#flash_notice').text
+		assert_equal(expected_text, actual_text)
+	end
+
+
+	def test_log_out
+	  	register_user({})
+	   	log_out
+	 	assert(@driver.find_element(:css, '.login').displayed?)
+	end
+
+
+	def test_log_in
+	   	user = register_user({})
+	   	log_out
+	   	log_in(user)
+
+	  	expected_login = user.login
+	  	actual_login = @driver.find_element(:css, '[class="user active"]').text
```

Comment:
this is not the best css locator
if this ```'.user'``` or this ```'.active'``` doesn't work you can use this one ```'.user.active'```